fastestimator/op/op.py

```diff
@@ -18,7 +18,7 @@
 import tensorflow as tf
 import torch
 
-from fastestimator.schedule import Scheduler
+from fastestimator.schedule.schedule import Scheduler
```

Comment:
schedule file is at same level as the schedule __init__, so we should import directly from fe.schedule

Reply:
yep,done